Enterprise-CMCS/macpro-mdct-mfp
# [MDCT-2958]: Have repeatedFields created with form

### Description



### Related ticket(s)


MDCT-2958

---
### How to test

Create a report
Go to Page 2 of State and Territory Initiatives or page 3
Use the modal and click yes to see 12 fields that are repeated from 1 wp.json field!
Check to see if they save
Check to see if they hydrate when opening the modal
Check to see if they clear if you swap to No in the choice list
Check to see if they populate the entitycard correctly

---
### Author checklist
